### DISCUSSION TITLE
Manual Installation on Wordpress file

### DISCUSSION THREAD

**Discussion by kenhehe** [halaxa/json-machine#53] [Q&A]
Hi, can you assist me on how I can use this without using docker. I'm currently working on a Wordpress file, and I needed to access JSON file which currently files size in 653MB

**Comment by halaxa** [comment]
JSON Machine does not need Docker to run. It is needed only for running JOSN Machine tests on all PHP platforms during its own development. Just install it using composer, and you should be good to go.
